testplan/sidebars.js

```diff
@@ -26,7 +26,13 @@ module.exports = {
 					'director/authentication/TCID-GUI-AUTH',
 					'director/authentication/TCID-GUI-CLUSTER',
 					'director/authentication/TCID-GUI-DASHBOARD',
-					'director/authentication/TCID-GUI-PROFILE'
+					'director/authentication/TCID-DIR-AUTH-LOCAL-ADMIN',
+					'director/authentication/TCID-DIR-AUTH-LOCAL-USER',
+					'director/authentication/TCID-DIR-AUTH-LOCAL-PERF',
+					'director/authentication/TCID-DIR-TEAMING-USER-INVITE',
```

Comment:
@bhaskarhc can you please remove teaming and opnebs tests from authentication section